second-gen/packages/swc/.storybook/preview.ts

```diff
@@ -14,6 +13,7 @@ setCustomElementsManifest(customElements);
 
 const preview = {
     parameters: {
+        layout: 'centered',
```

Comment:
I think centered or padded are nice customer experiences. The downside of making every component fullscreen is that the smaller components get squashed into the top left corner.